samples/core/model_serving_component/model_serving_component.ipynb

```diff
@@ -0,0 +1,297 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Copyright 2019 Google Inc. All Rights Reserved.\n",
+    "#\n",
+    "# Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "# you may not use this file except in compliance with the License.\n",
+    "# You may obtain a copy of the License at\n",
+    "#\n",
+    "#     http://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "# Unless required by applicable law or agreed to in writing, software\n",
+    "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "# See the License for the specific language governing permissions and\n",
+    "# limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Install Pipeline SDK - This only needs to be ran once in the enviroment. \n",
+    "!pip3 install kfp --upgrade\n",
+    "!pip3 install tensorflow==1.14"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## KubeFlow Pipelines Serving Component\n",
+    "In this notebook, we will demo:\n",
+    "\n",
+    "* Saving a Keras model in a format compatible with TF Serving\n",
+    "* Creating a pipeline to serve a trained model within a KubeFlow cluster\n",
+    "\n",
+    "Reference documentation:\n",
+    "* https://www.tensorflow.org/tfx/serving/architecture\n",
+    "* https://www.tensorflow.org/beta/guide/keras/saving_and_serializing\n",
+    "* https://www.kubeflow.org/docs/components/serving/tfserving_new/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Setup\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Set your output and project. !!!Must Do before you can proceed!!!\n",
+    "PROJECT_NAME =  'Your-Gcp-Project-ID'                          #'Your-GCP-Project-ID'\n",
+    "MODEL_NAME = 'MODEL_NAME'                                      # Model name matching TF_serve naming requirements       \n",
```

Comment:
nit: Can the MODEL_NAME be something like 'Model-Name' here? Just ask for consistency.

Reply:
good point! fixed.